src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -4,103 +4,74 @@ import java.net.URL
 import java.time.Instant
 import java.util.UUID
 
-import cats.Semigroup
-import cats.implicits._
 import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
 import akka.http.scaladsl.model.StatusCodes
+import cats.Semigroup
+import cats.implicits._
 import com.typesafe.config.ConfigFactory
 import net.ceedubs.ficus.Ficus._
-import org.broadinstitute.dsde.workbench.google.gcs.{GcsBucketName, GcsPath, GcsRelativePath}
-import org.broadinstitute.dsde.workbench.model.google.GoogleModelJsonSupport.GoogleProjectFormat
-import org.broadinstitute.dsde.workbench.leonardo.config.{ClusterDefaultsConfig, ClusterFilesConfig, ClusterResourcesConfig, DataprocConfig, ProxyConfig, SwaggerConfig}
-import org.broadinstitute.dsde.workbench.leonardo.model.ClusterStatus.ClusterStatus
-import org.broadinstitute.dsde.workbench.leonardo.model.StringValueClass.LabelMap
-import org.broadinstitute.dsde.workbench.model.WorkbenchEmail
+import org.broadinstitute.dsde.workbench.leonardo.config.{ClusterDefaultsConfig, ClusterFilesConfig, ClusterResourcesConfig, DataprocConfig, ProxyConfig}
+import org.broadinstitute.dsde.workbench.leonardo.model.Cluster._
+import org.broadinstitute.dsde.workbench.leonardo.model.google.ClusterStatus.ClusterStatus
+import org.broadinstitute.dsde.workbench.leonardo.model.google.GoogleJsonSupport._
+import org.broadinstitute.dsde.workbench.leonardo.model.google._
 import org.broadinstitute.dsde.workbench.model.WorkbenchIdentityJsonSupport._
-import org.broadinstitute.dsde.workbench.model.google.{GoogleProject, ServiceAccountKey}
-import spray.json.{DefaultJsonProtocol, DeserializationException, JsString, JsValue, JsonFormat, SerializationException}
-
-import scala.language.implicitConversions
-
-case class NegativeIntegerArgumentInClusterRequestException()
-  extends LeoException(s"Your cluster request should not have negative integer values. Please revise your request and submit again.", StatusCodes.BadRequest)
-
-case class OneWorkerSpecifiedInClusterRequestException()
-  extends LeoException("Google Dataproc does not support clusters with 1 non-preemptible worker. Must be 0, 2 or more.")
-
+import org.broadinstitute.dsde.workbench.model.google.GoogleModelJsonSupport._
+import org.broadinstitute.dsde.workbench.model.google._
+import org.broadinstitute.dsde.workbench.model._
+import spray.json._
 
-// this needs to be a Universal Trait to enable mixin with Value Classes
-// it only serves as a marker for StringValueClassFormat
-sealed trait StringValueClass extends Any
-case class IP(string: String) extends AnyVal with StringValueClass
-case class ZoneUri(string: String) extends AnyVal with StringValueClass
-
-// productPrefix makes toString = e.g. "Cluster (clustername)"
-
-case class ClusterName(string: String) extends AnyVal with StringValueClass {
-  override def productPrefix: String = "Cluster "
-}
+// Create cluster API request
```

Comment:
Really just a lot of moving things around in this file, not really any changes to the classes themselves.